app/controllers/api/v1/invoices_controller.rb

```diff
@@ -281,14 +291,12 @@ def preview_params
             :tax_identification_number,
             :currency,
             :timezone,
-            shipping_address: [
```

Comment:
Don't we want to keep `shipping_address` as well? Please take. look at this file: https://github.com/getlago/lago-api/blob/main/app/services/integrations/aggregator/taxes/invoices/payload.rb